pkgs/data/fonts/orbitron/default.nix

```diff
@@ -0,0 +1,46 @@
+{ stdenv, lib, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  name = "orbitron-${version}";
+  version = "20110526";
+
+  src = fetchFromGitHub {
+    owner  = "theleagueof";
+    repo   = "orbitron";
+    rev    = "13e6a52";
+    sha256 = "1c6jb7ayr07j1pbnzf3jxng9x9bbqp3zydf8mqdw9ifln1b4ycyf";
+  };
+
+  installPhase = ''
+    otfdir=$out/share/fonts/opentype/orbitron
+    ttfdir=$out/share/fonts/ttf/orbitron
+    mkdir -p $otfdir $ttfdir
+    cp -v Orbitron*.otf $otfdir 
+    cp -v Orbitron*.ttf $ttfdir
+  '';
+
+  meta = with lib; {
+    homepage = "https://www.theleagueofmoveabletype.com/orbitron";
+    downloadPage = "https://www.theleagueofmoveabletype.com/orbitron/download";
+    description = ''
```

Comment:
The `longDescription` attribute is more appropriate for long-form descriptions; `description` is for short descriptions only.